src/plugins/profiling/server/routes/search_topNHosts.ts

```diff
@@ -8,11 +8,11 @@
 import type { DataRequestHandlerContext } from '../../../data/server';
 import type { IRouter } from '../../../../core/server';
 import { getRemoteRoutePaths } from '../../common';
-import { queryTopNCommon } from "./search_topN";
+import { queryTopNCommon } from './search_topN';
 
 export function registerTraceEventsTopNHostsSearchRoute(
   router: IRouter<DataRequestHandlerContext>
 ) {
   const paths = getRemoteRoutePaths();
-  return queryTopNCommon(router, paths.TopNHosts, 'HostName');
+  return queryTopNCommon(router, paths.TopNHosts, 'HostID');
```

Comment:
This is the fix, all other changes are just cleanups / cosmetics.
In general, it would be nicer to split these things up into two PRs (faster to review).
In this case it was quite easy, but today I had to review a prodfiler PR where larger (non-functional) refactorings where mixed with other functional changes - that was quite a mess to read.
TY 😄

Reply:
I split the functional and refactoring parts in 2 separate commits, and wrote in the PR description about the additional linting.
Anyway you have a valid point and will apply a stricter separation when bigger changes are going to be rolled out 🙏🏼